gwdetchar/omega/html.py

```diff
@@ -393,7 +391,7 @@ def toggle_link(plottype, channel, pranges):
     chanstring = channel.name.replace('-', '_').replace(':', '-')
     captions = [p.caption for p in channel.plots[plottype]]
     return markup.oneliner.a(
-        '<b>%s</b>' % text, class_='dropdown-item',
+        '%s' % text, class_='dropdown-item',
```

Comment:
@alurban, can you replace `'%s' % text` with just `text` (since `text` is already a `str`). There's a few more of those below as well.